content/release-notes/1.19.0.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2020-09-14
+linktitle: "1.19.0"
+title: "1.19.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+  * Support for airgapped [headless installs](/kotsadm/installing/automating/).
+  * Support resumable uploads for airgap bundles in Kotsadm UI.
+  * Added support for additional authentications methods including OIDC and IBM Cloud to `kubectl kots`.
+  * Added support for troubleshoot.sh/v1beta2 SupportBundle specs.
```

Comment:
do we need a link here?

Reply:
Also, these are two somewhat separate things. Version, and kind: SupportBundle.

Reply:
```suggestion
  * Support for `avpiVersion: troubleshoot.sh/v1beta2`
  * Support for `kind: SupportBundle`.
```